app/assets/stylesheets/darkswarm/_shop-inputs.css.scss

```diff
@@ -7,6 +7,9 @@
   // #search
   @include placeholder(rgba(0, 0, 0, 0.4), #777);
 
+  padding-left: 1rem;
+  padding-right: 1rem;
+
```

Comment:
Interesting...

Is there a reason you are changing the style on such a high level class? Have you considered impact on other parts of the app? Was this intended? If not, I'd go for a more specific class. This is the form you want to fix right?
https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/views/spree/users/edit.html.haml
Maybe you can add a class here, maybe ".darkswarm.user-form" and then you can style it (I am not sure what's the best css file to do it, maybe a new one?)

Also, have you considered different screen sizes? Your change would conflict with this for example, right?
https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/assets/stylesheets/darkswarm/_shop-inputs.css.scss#L41 

I think @kristinalim will be the best person to advise you on this PR. Could you pls review this PR @kristinalim ?

Reply:
I can definitely move this to a more specific level. It was not my intent to impact other parts of the app; I was only concerned with how it affected edit.html.haml specifically. I'll go ahead and add a new css class to deal with the issue.